src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1547,6 +1547,9 @@ def process_vmss_create_namespace(cmd, namespace):
             if value is not None:
                 raise ArgumentUsageError(f'usage error: {param} is not supported for Flex mode')
 
+        if namespace.vm_sku and not namespace.image:
```

Comment:
Do we need handle the case `if namespace.image and not namespace.vm_sku`?

Reply:
It has been confirmed with @fitzgeraldsteele  that for creating Flex VMSS, we only need to raise the usage error 
> raise ArgumentUsageError('usage error: please specify the --image when you want to specify the VM SKU')

Because

> For creating VMSS, the information of VM profile and SKU should be either passed in or not passed in. Therefore, if users pass in the --vm-sku parameter, CLI needs to verify that users must also pass in the --image parameter